Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -152,6 +157,7 @@
       - id: AccessConfigurator
       - id: RCD
       - id: RCDAmmo
+      - id: HandheldPanicButton
```

Comment:
are some of these added 2x once to the hardsuit locker and again to the CE locker again? 
      essentially duplicates?